internal/agent/install.go

```diff
@@ -102,11 +102,13 @@ func Install(dir ...string) error {
 		}
 	})
 
-	// Try to pull userdata once more. best-effort
-	if _, err := os.Stat("/oem/userdata"); err != nil {
-		if err := machine.ExecuteCloudConfig("/system/oem/00_datasource.yaml", "rootfs.before"); err != nil {
-			fmt.Println("Warning: Failed pulling from datasources")
+	for {
```

Comment:
this is looking good! maybe we should timeout here and not let the loop go forever?

Reply:
That is a good idea!